testsuite/python_tools/complex/test_python_tools_complex.py

```diff
@@ -21,7 +40,7 @@ def setUp(self):
         system.bc = espressopp.bc.OrthorhombicBC(system.rng, box)
         system.skin = 0.3
         system.comm = MPI.COMM_WORLD
-        nodeGrid = espressopp.tools.decomp.nodeGrid(espressopp.MPI.COMM_WORLD.size)
+        nodeGrid = espressopp.tools.decomp.nodeGrid(box,1.5,0.3,espressopp.MPI.COMM_WORLD.size)
```

Comment:
Please use `system.skin` instead of `0.3`.

Reply:
sure